src/python/lib/ensembl/compara/hal/__init__.py

```diff
@@ -0,0 +1,15 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module for working with HAL alignment data."""
```

Comment:
```suggestion
"""Module for working with HAL alignment data."""

from .maf_to_fasta import *
```
This piece missing explains the need for `# pylint: disable-next=no-member` before calling `convert_maf_to_fasta` function.

Reply:
I was definitely off my game yesterday :facepalm: This change is not for your unit test, because there you import the `maf_to_fasta` module **directly**. However, you still want users to be able to access `convert_maf_to_fasta()` function if they `from ensembl.compara.hal import *` (or `from ensembl.compara import hal`, IIRC) but not `SeqIO` as an element of your module: `__all__` defines the list of symbols imported when using the star (`*`) import, which can include what you import from other libraries (but this can cause a name conflict when misused).

Reply:
Fair point. I've applied this change in [commit 82f4cf4](https://github.com/Ensembl/ensembl-compara/pull/406/commits/82f4cf4c5f8fc15a162ed015f24368b3701614e7).